src/main/java/pl/mimuw/worker/service/WorkerService.java

```diff
@@ -73,20 +80,14 @@ private void processMessage(final ConvertedAcknowledgeablePubsubMessage<MonitorT
     }
 
     @SneakyThrows
-    public void monitorService(final MonitorTaskMessage monitorTask) {
-        for (long currentTimeSecs = System.currentTimeMillis() / 1000L;
+    private void monitorService(final MonitorTaskMessage monitorTask) {
+        for (int currentTimeSecs = currentTimeSecs();
              currentTimeSecs < monitorTask.getTaskDeadlineTimestampSecs();
-             currentTimeSecs = System.currentTimeMillis() / 1000L) {
+             currentTimeSecs = currentTimeSecs()) {
             log.info("Pinging service: {}, time: {}", monitorTask.getServiceUrl(), currentDate());
```

Comment:
Remember to remove it later because it will spam our logs

Reply:
i will turn it off by application.properties if needed - lets leave it for now